WooCommerce/Classes/Authentication/Prologue/LoginOnboardingViewController.swift

```diff
@@ -0,0 +1,142 @@
+        bottomView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(bottomView)
+        NSLayoutConstraint.activate([
+            bottomView.leadingAnchor.constraint(equalTo: view.safeLeadingAnchor),
+            bottomView.trailingAnchor.constraint(equalTo: view.safeTrailingAnchor),
+            bottomView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 0.3),
+            bottomView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+
+    func configureCurvedImageView() {
+        let imageView = UIImageView(image: .curvedRectangle.withRenderingMode(.alwaysTemplate))
+        imageView.tintColor = .authPrologueBottomBackgroundColor
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(imageView)
+        NSLayoutConstraint.activate([
+            imageView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            imageView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            imageView.topAnchor.constraint(equalTo: view.safeTopAnchor, constant: 52),
+            imageView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 0.7)
+        ])
+    }
+
+    func configureWooLogo() {
+        let imageView = UIImageView(image: .wooLogoPrologueImage)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(imageView)
+        NSLayoutConstraint.activate([
+            imageView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            imageView.widthAnchor.constraint(equalToConstant: 176),
+            imageView.heightAnchor.constraint(equalToConstant: 36),
+            imageView.topAnchor.constraint(equalTo: view.safeTopAnchor, constant: 28)
+        ])
+    }
+
+    func configureStackView() {
+        configureStackViewSubviews()
+
+        stackView.axis = .vertical
+        stackView.spacing = Constants.stackViewSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(stackView)
+        view.pinSubviewToSafeArea(stackView, insets: Constants.stackViewInsets)
+    }
+
+    func configureStackViewSubviews() {
+        let carousel = createFeatureCarousel()
+
+        let continueButton = createContinueButton()
+        let skipButton = createSkipButton()
+        let buttonStackView: UIStackView = {
+            let stackView = UIStackView(arrangedSubviews: [continueButton, skipButton])
+            stackView.spacing = Constants.buttonStackViewSpacing
+            stackView.axis = .vertical
+            return stackView
+        }()
+
+        stackView.addArrangedSubviews([carousel, buttonStackView])
+    }
+
+    func createFeatureCarousel() -> UIView {
+        pageViewController.view.translatesAutoresizingMaskIntoConstraints = false
+
+        addChild(pageViewController)
+        return pageViewController.view
+    }
+
+    func createContinueButton() -> UIButton {
```

Comment:
Should we change the naming as `next` instead of `continue` considering that the button is labeled as "Next"?

Reply:
Yea makes sense, updated in https://github.com/woocommerce/woocommerce-ios/pull/7291/commits/c7dd9ab4d249c7c36cd770a2275570c43fa9f76d